### PR TITLE
OSDOCS-9827: adds two-version update release note MicroShift

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.16

Issue:
[OSDOCS-9827](https://issues.redhat.com/browse/OSDOCS-9827)

Link to docs preview:
[Updates Y+2](https://73997--ocpdocs-pr.netlify.app/microshift/latest/microshift_release_notes/microshift-4-16-release-notes#microshift-4-16-updating)

QE review:
- [x] QE has approved this change.

Additional information:
Feature PR https://github.com/openshift/openshift-docs/pull/73988

### CODE REVIEW

File: microshift_release_notes/microshift-4-16-release-notes.adoc [hunk 1]
```diff
@@ -41,15 +41,18 @@ This release adds improvements related to the following components and concepts.
...
-Updates for both minor releases and patch releases are supported.
+Updating from one minor version to another two minor versions away in a single step is now supported. Updates for both single-version minor releases and patch releases are also still supported.
 
 [id="microshift-4-16-updates-supported"]
-==== The following updates are supported
-The following list provides update details:
+==== Update two minor versions away in a single step
+See the following list for details:
 
+* You can update from one long-life-cycle version of {microshift-short} directly to another, without applying the intermediate version. For example, you can update directly to 4.16 from 4.14.
```
Comment (dhensel-rh) [inline microshift_release_notes/microshift-4-16-release-notes.adoc:50]:
```suggestion
* You can update from one long-life-cycle version of {microshift-short} directly to another, without applying the intermediate version. For example, you can update directly from 4.14 to 4.16.
```

File: microshift_release_notes/microshift-4-16-release-notes.adoc [hunk 2]
```diff
@@ -41,15 +41,18 @@ This release adds improvements related to the following components and concepts.
 
 [id="microshift-4-16-updating"]
 === Updating
-Updates for both minor releases and patch releases are supported.
+Updating from one minor version to another two minor versions away in a single step is now supported. Updates for both single-version minor releases and patch releases are also still supported.
 
 [id="microshift-4-16-updates-supported"]
-==== The following updates are supported
-The following list provides update details:
+==== Update two minor versions away in a single step
```
Comment (dhensel-rh) [inline microshift_release_notes/microshift-4-16-release-notes.adoc:47]:
```suggestion
==== Update two minor versions away (Y+2) in a single step 
```

File: microshift_release_notes/microshift-4-16-release-notes.adoc [hunk 3]
```diff
@@ -18,7 +18,7 @@ Version 4.16 of {product-title} includes new features and enhancements. {microsh
 You can deploy {microshift-short} clusters to on-premise, cloud, disconnected, or offline environments.
 
 //TODO: Update OP system versions
-{microshift-short} {product-version} is supported on {op-system-ostree-first} and {op-system-base-full} 9.2 and 9.3.
+{microshift-short} {product-version} is supported on {op-system-base-full} and {op-system-ostree-first} and 9.4.
```
Comment (dhensel-rh) [inline microshift_release_notes/microshift-4-16-release-notes.adoc:21]: Is it redundant to mention only MicroShift 4.16 (and above) is supported on RHEL 9.4 ? Previous versions of MicroShift are not.

File: microshift_release_notes/microshift-4-16-release-notes.adoc (hunk 2 above)
Comment (ShaunaDiaz) [reply microshift_release_notes/microshift-4-16-release-notes.adoc:47]: Not docs style.

File: microshift_release_notes/microshift-4-16-release-notes.adoc (hunk 3 above)
Comment (ShaunaDiaz) [reply microshift_release_notes/microshift-4-16-release-notes.adoc:21]: The release notes are only for 4.16. This is standard text in all of the version-specific release notes.

Comment (dhensel-rh) [reply microshift_release_notes/microshift-4-16-release-notes.adoc]: Thank you for the clarification.

**Comment by ShaunaDiaz** [comment]
QE pending

**Comment by ShaunaDiaz** [comment]
@dhellmann Could you PTAL?

**Review by dhellmann — COMMENTED** [review]
/lgtm

**Comment by dhensel-rh** [comment]
label qe-approved

File: microshift_release_notes/microshift-4-16-release-notes.adoc [hunk 4]
```diff
@@ -41,15 +41,18 @@ This release adds improvements related to the following components and concepts.
 
 [id="microshift-4-16-updating"]
 === Updating
-Updates for both minor releases and patch releases are supported.
+Updating from one minor version to another two minor versions away in a single step is now supported. Updates for both single-version minor releases and patch releases are also still supported.
```
Comment (mburke5678) [inline microshift_release_notes/microshift-4-16-release-notes.adoc:44]: Does this work? It's a bit tighter. 
```suggestion
Updating  two minor versions in a single step is now supported. Updates for both single-version minor releases and patch releases are also still supported.
```

File: microshift_release_notes/microshift-4-16-release-notes.adoc [hunk 5]
```diff
@@ -41,15 +41,18 @@ This release adds improvements related to the following components and concepts.
...
 
 [id="microshift-4-16-updates-supported"]
-==== The following updates are supported
-The following list provides update details:
+==== Update two minor versions away in a single step
+See the following list for details:
 
+* You can update from one long-life-cycle version of {microshift-short} directly to another, without applying the intermediate version. For example, you can update directly to 4.16 from 4.14. Updating 4.14 to 4.15 before updating to 4.16 is no longer required.
```
Comment (mburke5678) [inline microshift_release_notes/microshift-4-16-release-notes.adoc:50]: The other PR, and most of this one, uses lower-to-higher versioning.
```suggestion
* You can update from one long-life-cycle version of {microshift-short} directly to another, without applying the intermediate version. For example, you can update directly from 4.14 to 4.16. Updating 4.14 to 4.15 before updating to 4.16 is no longer required.
```

**Comment by mburke5678** [comment]
@ShaunaDiaz A few nits. Otherwise LGTM

**Pull request merged**